src/main/webapp/resources/js/apis/users/users.js

```diff
@@ -11,7 +51,7 @@ const BASE_URL = setBaseUrl(`ajax/users/edit`);
  */
 export async function setUsersDisabledStatus({ isEnabled, id }) {
```

Comment:
This can probably be added to the api above as a mutation

Reply:
Please see fa224218b92b6deae508fb161633f5fec3ca2855 and 82efb4226a322c5bd1fc3c9069e53ed636eaf1e4.